Gems/LmbrCentral/Code/Source/Shape/EditorReferenceShapeComponent.h

```diff
@@ -8,24 +8,24 @@
 
 #pragma once
 
-#include <Components/ReferenceShapeComponent.h>
-#include <Vegetation/Editor/EditorVegetationComponentBase.h>
+#include<LmbrCentral/Component/EditorWrappedComponentBase.h>
+#include <Shape/ReferenceShapeComponent.h>
 
-namespace Vegetation
+namespace LmbrCentral
 {
     class EditorReferenceShapeComponent
-        : public EditorVegetationComponentBase<ReferenceShapeComponent, ReferenceShapeConfig>
+        : public EditorWrappedComponentBase<ReferenceShapeComponent, ReferenceShapeConfig>
     {
     public:
-        using BaseClassType = EditorVegetationComponentBase<ReferenceShapeComponent, ReferenceShapeConfig>;
+        using BaseClassType = EditorWrappedComponentBase<ReferenceShapeComponent, ReferenceShapeConfig>;
         AZ_EDITOR_COMPONENT(EditorReferenceShapeComponent, EditorReferenceShapeComponentTypeId, BaseClassType);
         static void Reflect(AZ::ReflectContext* context);
 
-        static constexpr const char* const s_categoryName = "Vegetation";
-        static constexpr const char* const s_componentName = "Vegetation Reference Shape";
+        static constexpr const char* const s_categoryName = "Shape";
+        static constexpr const char* const s_componentName = "Reference Shape";
         static constexpr const char* const s_componentDescription = "Enables the entity to reference and reuse shape entities";
         static constexpr const char* const s_icon = "Editor/Icons/Components/Vegetation.svg";
```

Comment:
This icon should also get replaced.

Reply:
Is there one available?

Reply:
I guess just use Editor/Icons/Components/Component_Placeholder.svg to match what you're using on the next line for the viewport icon.  Longer term we'll need to request an icon for this component.